### ISSUE TITLE
Kitematic container list synchronization

### ISSUE THREAD

**Issue by ronnyek** [docker/kitematic#1893]
### Expected behavior

Updating containers, installing and running docker-compose up frequently results in kitematic automatically updating the list of containers.
### Actual behavior

Only occasionally does it not. Seems like frequently I need to trigger a "refresh" of sorts by trying to do things with other containers that are active. 
### Information about the Issue

I don't know if this is temporary problem... know that its intermittent and half wondered if it'd just make sense to have a refresh button or something.

This issue is docker for windows, kitematic windows latest afaik.
### Steps to reproduce the behavior
1. Open Kitematic user interface
2. From console, run any operation that will add/remove or otherwise change state of docker containers
3. Frequently the list gets out of sync, and will eventually get back in sync, but only after other operations... (deleting another container will cause it to re-query container list)

**Comment by FrenchBen** [comment]
I've seen it happen, but never been able to figure out why the event isn't propagated. 

To help with the refresh, we added a few release back a CMD+R, which is also available from the Menu, under View -> Refresh Container List.

If you feel like debugging why the events aren't propagated properly, it would be of great help!

**Comment by clemens-tolboom** [comment]
Good to know __CMD+R__ is available. I tried right click for a menu.

**Comment by anton-arsentyev** [comment]
@FrenchBen may be adding refresh button would be great idea ;)
can try to make such PR

**Comment by FrenchBen** [comment]
@anton-arsentyev the refresh exists under the menu and by clicking the 'container list' - Is there a real need to have the button added as well?

**Comment by anton-arsentyev** [comment]
@FrenchBen think not for container list but for local images list, I read this issue wrong sorry)

**Comment by clemens-tolboom** [comment]
I disagree with _won't fix_ as the reporter refers to console actions not reflected back in the list of containers.

This needs at least
- proper labels: bug, won't fix
- better title reflecting the behaviour: "Kitematic container list incomplete" to prevent more report.
